app/src/org/commcare/tasks/FormRecordLoaderTask.java

```diff
@@ -150,7 +151,10 @@ protected Integer doInBackground(FormRecord... params) {
         // Get the date in a searchable format.
         recordTextDesc.add(DateUtils.formatDateTime(context, current.lastModified().getTime(), DateUtils.FORMAT_NO_MONTH_DAY | DateUtils.FORMAT_NO_YEAR).toLowerCase());

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/33f1713b6eeb69538a3a8241632b7fbd686a3ef3/app/src/org/commcare/tasks/FormRecordLoaderTask.java#L152"></span>Implicitly using the default locale is a common source of bugs: Use <code>toLowerCase(Locale)</code> instead. For strings meant to be internal use <code>Locale.ROOT</code>, otherwise <code>Locale.getDefault()</code>.</td>
    </tr>
  </tbody>
</table>